README.md

```diff
@@ -19,3 +19,21 @@ Documents related to Code for Poznan organization.
 ## License
 
 Documents are licensed under [CC-BY-SA-4.0-PL](https://creativecommons.org/licenses/by-sa/4.0/deed.pl) (with the exception of Code for Poznan logo).
+
+## Welcome message - mail and messenger
+
+Cześć *Tu wpisz imię* !
+
+Kwarantanna nadal trwa i nie wiadomo kiedy wszystko wróci do normy, dlatego kontynuujemy nasze spotkania w wersji online.
```

Comment:
Teraz już chyba nie jest to kwarantanna, może nazwiemy to inaczej? 

Moze:
* Jesteśmy odpowiedzialną organizacją i póki wszystko nie wróci do normy, kontynuujemy nasze spotkania w wersji online.

Reply:
'odpowiedzialna organizacja' brzmi... dziwnie :)
napiszmy może po prostu, że z powodu koronawirusa kontynuujemy nasze spotkania w wersji online.

Reply:
`W związku z panującą pandemią, postanowiliśmy kontynuować nasze spotkania w wersji online.`

Reply:
Wersja Tomka jest w porządku: `W związku z panującą pandemią postanowiliśmy kontynuować nasze spotkania w wersji online.`, (ale bez przecinka w środku).